### ISSUE TITLE
created date and updated date in metadata

### ISSUE THREAD

**Issue by copdips** [Guts/mkdocs-rss-plugin#229]
Hello,

I have both dates under the `date` key in the yaml metadata, could you please provide the syntax to show the created date ?

becase with current syntax:

```yaml
date_from_meta:
    as_creation: date
```

I got error:
`[rss-plugin] Creation date of xxx.md has not been recognized.`

https://squidfunk.github.io/mkdocs-material/plugins/blog/?h=created#meta.date

![image](https://github.com/Guts/mkdocs-rss-plugin/assets/13083856/d86759d1-7caf-42da-a75a-53289afba0f2)

**Comment by copdips** [comment]
okey, I found it, it should be : 

```yaml
      date_from_meta:
        as_creation: date.created
```

**Comment by copdips** [comment]
Hello,

I reopened this issue, as some of my posts use:

```yaml
date:
   created:  2023-01-31
   updated: 2023-02-21
```

and some others use:

```yaml
date: 2023-01-31
```

is there anyway to handle both of it ?

**Comment by Guts** [comment]
Hello,

Thanks for your interest and taking time to ask here.

It's not possible to use both for now and dealing with heterogenous date/time formats is quite tricky and really hard to maintain.

In one of my project, I'm using a Mkdocs hook to check date format  to warn the author or autofix it if possible.

**Comment by Guts** [comment]
Additional note: you can specify the format you want the RSS plugin to consider as valid with `datetime_format` option.

**Comment by copdips** [comment]
thx, I will fix that to have one standard everywhere

**Comment by copdips** [comment]
Hello @Guts 

I reopen this issue, as it seems that file git log date is always used by `pubData` even `date_from_meta` is specified.

In my mkdocs.yml, I have:

```yaml
      date_from_meta:
        as_creation: date.created
```

But when I checked `/feed_rss_created.xml`, it always uses file git log date instead of `date.created` in the file front matter.

**Comment by Guts** [comment]
Released as part of 1.11.1: https://github.com/Guts/mkdocs-rss-plugin/discussions/238
